ci/test_integration_espnetez.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source tools/activate_python.sh
+PYTHONPATH="${PYTHONPATH:-}:$(pwd)/tools/s3prl"
+export PYTHONPATH
+python="python -m coverage run --append"
+cwd=$(pwd)
+
+gen_dummy_coverage(){
+    # To avoid a problem when parallel running for `coverage run`.
+    # Please put this command after cd ./egs2/foo/bar
+    touch empty.py; ${python} empty.py
+}
+
+#### Make sure chainer-independent ####
+python3 -m pip uninstall -y chainer
+
+# Download mini_an4 as test data and prepare flac data
+cd ./egs2/mini_an4/asr1
+./run.sh --stage 1 --stop-stage 1
+./run.sh --stage 2 --stop-stage 4 --feats-type "raw"
+cp -r dump/raw data/
+./run.sh --stage 2 --stop-stage 4 --feats-type "raw_copy" \
+    --train_set raw/train_nodev --valid_set raw/train_dev --test_sets raw/test
```

Comment:
Why do we need to create dump directory twice?

Reply:
I refer to the espnet2 test to write this part, but will check if using the only line 23 is enough.